docs/jailer.md

```diff
@@ -277,6 +277,13 @@ Note: default value for `<api-sock>` is `/run/firecracker.socket`.
   associated with `--daemonize` runs towards the end, instead of the very
   beginning. We are working on adding better logging capabilities.
 
+### Notes
+
+- The recommended method to get Firecracker process's pid is to use
+  `--new-pid-ns` flag and read Firecracker's pid from the `firecracker.pid` file
+  present in the jailer's root directory. This is also the recommended method to
```

Comment:
It feels like there is some repetition in the second sentence. It looks like we want to convey 2 things:
- use `--new-pid-ns`
- use the pidfile

Not sure we need to mention the latter twice.

Reply:
pid could be used for other stuff as well so in 1st line I just mentioned the way to get it and 2nd line focused on the specific use case we wanted to highlight.

Reply:
renamed the section and removed the recommendation to kill firecracker. Since we mention the change in behaviour I don't think we need to explicitly suggest what should be done to kill firecracker.
If we work on making the firecracker pid always available in the file then I think we can have a separate section in Notes to suggest a way to kill firecracker.